test/e2e/v2/petstore.js

```diff
@@ -55,9 +55,15 @@ describe('swagger 2.0 specification tests using petstore.json', function () {
       return driver.wait(until.elementLocated(locator), testTimeout);
     });
   });
+  // nesting file inputs in labels causes the file upload to fail
+  it('should have not have a file input inside a label', function () {
+    return driver.findElements(webdriver.By.css('label input[type="file"]')).then((i)=> {
+      expect(i).to.be.empty;
+    })
+  })
 
-  after((done) => {
-    servers.close();
-    done();
+  after(() => {
+    servers.close()
+    return driver.quit()
```

Comment:
Oh, ok. Thanks for explaining just now how `driver` works and that the need to either return a callback or execute a callback comes from chair. SGTM!